src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -492,6 +497,7 @@ QVariantHash AutoDownloadRule::toLegacyDict() const
         {"category_assigned", assignedCategory()},
         {"use_regex", useRegex()},
         {"add_paused", triStateBoolToAddPausedLegacy(addPaused())},
+        {"create_subfolder", triStateBoolToAddPausedLegacy(createSubfolder())},
```

Comment:
You shouldn't change to/fromLegacyDict methods. They exist to support import/export rules into legacy file format for old qBittorrent versions which cannot understand this new field.